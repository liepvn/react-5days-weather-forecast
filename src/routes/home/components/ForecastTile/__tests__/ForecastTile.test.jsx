import { render } from 'test/test-utils';

import { ForecastTile } from '..';

describe('ForecastTile', () => {
  it('should render properly', () => {
    const forecast = {
      maxTemp: 31.075000000000003,
      minTemp: 25.095,
      date: '2021-08-15',
      weatherStateAbbr: 'hr',
      weatherStateImage: 'https://www.metaweather.com/static/img/weather/png/64/hr.png',
      humidity: 76,
      temp: 30.259999999999998
    };

    const { container } = render(<ForecastTile forecast={forecast} />);

    expect(container).toMatchSnapshot();
  })
});