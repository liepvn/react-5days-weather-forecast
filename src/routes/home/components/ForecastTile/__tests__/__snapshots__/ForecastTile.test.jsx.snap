// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastTile should render properly 1`] = `
<div>
  <div
    class="forecast-tile mb-2 border rounded"
  >
    <div
      class="row"
    >
      <div
        class="col-3"
      >
        <div
          class="d-flex flex-column"
        >
          <p
            class="mb-0 text-uppercase fs-3"
          >
            Sun
          </p>
          <div
            class="humidity"
          >
            <div>
              <i
                class="bi bi-droplet-fill"
              />
               
              <span
                class="text-muted"
              >
                76
                %
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        class="col-3"
      >
        <img
          class="w-image"
          src="https://www.metaweather.com/static/img/weather/png/64/hr.png"
        />
      </div>
      <div
        class="d-flex justify-content-end col-6"
      >
        <div
          class="temperature d-flex flex-row"
        >
          <div
            class="temperature__value d-flex flex-column text-end"
          >
            <div>
              <span
                class="fs-3"
              >
                30°C
              </span>
            </div>
            <div
              class="text-muted"
            >
              <span>
                25°C
              </span>
               -
               
              <span>
                31°C
              </span>
            </div>
          </div>
          <div
            class="temperature__icon align-self-center"
          >
            <i
              class="bi bi-thermometer-sun fs-1"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
